week-4/Homework/mandatory/4-groceries.js

```diff
@@ -51,12 +62,20 @@ Exercise 2:
   Finally use console.log() to print out the Object.
 */
 // Gather weekend item names into this array
+
 let numberOfItemsPerWeak = {
   monday: 0,
   tuesday: 0,
   wednesday: 0,
   thursday: 0,
   fridray: 0,
   saturday: 0,
-  sunday: 0
+  sunday: 0,
 };
+
+numberOfItemsPerWeak = [];
+
+for (key in weeklyMealPlan) {
+  numberOfItemsPerWeak.push(`${key}: ${weeklyMealPlan[key].length}`);
+}
```

Comment:
here we don't need the key because we already have them, we just need the length of each item as the value. Fixed that, please.